Automattic/wp-calypso
# My Home: update learn and grow illustrations for consistency

#### Changes proposed in this Pull Request
This PR creates more consistency between the illustrations under the Learn and Grow section of My Home.

* Add a wireframe-style illustration for promoting Earn features.
* Remove drop shadow from the Gutenberg illustration.
* Update photo library illustration so it fits in the card better.
* Update the file naming for clearer organization.

**Before**
<img width="715" alt="Screen Shot 2020-06-03 at 10 48 46 AM" src="https://user-images.githubusercontent.com/448298/83652342-80026f80-a588-11ea-919c-fae446450dc6.png">

**After**
<img width="717" alt="Screen Shot 2020-06-03 at 10 46 17 AM" src="https://user-images.githubusercontent.com/448298/83652353-8395f680-a588-11ea-987a-554b2d676c15.png">


#### Testing instructions

* Check out this branch.
* Patch your sandbox with D44038-code.
* Sandbox the API.
* Visit My Home on any site with the default view (no task cards).
* Confirm the illustrations appear like the screenshots above.
